website/docs/intro/install/index.mdx

```diff
@@ -0,0 +1,58 @@
+---
+sidebar_position: 1
+description: |-
+  Learn how to install OpenTofu.
+---
+
+import DocCardList from "@theme/DocCardList";
+
+# Installing OpenTofu
+
+You can install OpenTofu via a wide range of methods. Please select your operating system and installation method:
+
+<DocCardList
```

Comment:
**(praise)** It's a brilliant idea - nicely done! 🎉

Reply:
Thank you, glad you like it. The DocCardList stylings really do come in handy for these summary pages, we should do more of those.

Reply:
Definitely!